iphone/Classes/TiUIListView.m

```diff
@@ -578,10 +578,10 @@ - (void)updateSearchResults:(id)unused
 {
     if (searchActive) {
         [self buildResultsForSearchText];
-        [resultViewController.tableView reloadData];
     }
-    else {
-        [_tableView reloadData];
+    [_tableView reloadData];
+    if ([searchController isActive]) {
+        [resultViewController.tableView reloadData];
```

Comment:
there are 2 ways we can use search in list view -
1. add searcbar on window, and show search result in same table view (see last test case in ticket)
2.  make search bar part of list view  (see test case in ticket except last)
In second way, when we click on searchbar [searchController isActive] will be yes.
But for searchActive being to Yes, there should be atleast one character in search bar.
So when second way of search used, user clicked on search bar and not typed any character, [searchController isActive] will be YES and searchActive will be NO.
If you see the old implementation, it is in same way.

Are you sure there is problem in indentation, I don't think so.